File: Frontend/src/app/entidades/educacion.ts
export interface Educacion{
    id:number;
    titulo:string;
    institucion:string;
    anio_inicio:number;
    anio_egreso:number;
    

    constructor(id:number, nombre:string, apellido:string, avatar:string, descripcion:string, posicion:string)
    
    {
        this.id=id;
        this.nombre=nombre;
        this.apellido=apellido;
        this.avatar=avatar;
        this.descripcion=descripcion;
        this.posicion=posicion;
    
        
    }
}

File: Backend/michi/src/main/java/com/portfolioAP/michi/Model/Educacion.java
package com.portfolioAP.michi.Model;

import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@Entity
public class Educacion {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;
    @Basic
    String titulo;
    String institucion;
    Number anio_inicio;
    Number anio_egreso;
    
   
    public Educacion(Long id, String titulo, String institucion, Number anio_inicio, Number anio_egreso) {
       this.id = id;
       this.titulo = titulo;
       this.institucion = institucion;
       this.anio_inicio = anio_inicio;
       this.anio_egreso = anio_egreso;
        
    }
    
    public Educacion() {
        
    }

}